specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2958 @@
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The requested test run is stopped. Call GET /testruns/{testRunId} to check the status.",
+            "schema": {
+              "$ref": "#/definitions/TestRunModel20221101"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Stop a test run": {
+            "$ref": "./examples/StopTestRun.json"
+          }
+        }
+      }
+    },
+    "/testRuns/{testRunId}/metricNamespaces": {
+      "get": {
+        "tags": [
+          "TestRun"
+        ],
+        "summary": "Lists the metric namespaces for a load test run.",
+        "operationId": "TestRun_MetricNamespaces",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "testRunId",
+            "description": "Unique name for the load test run, must contain only lower-case alphabetic, numeric, underscore or hyphen characters.",
+            "required": true,
+            "type": "string",
+            "maxLength": 50,
+            "minLength": 2,
+            "pattern": "^[a-z0-9_-]*$"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get the list of metric namespaces",
+            "schema": {
+              "$ref": "#/definitions/MetricNamespaceCollection"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Lists the metric namespaces for a load test run.": {
+            "$ref": "./examples/GetTestRunMetricsNamespaces.json"
+          }
+        }
+      }
+    },
+    "/testRuns/{testRunId}/metricDefinitions": {
```

Comment:
Note inconsistent casing here: "testRuns" vs "testruns". This reinforces the earlier comment that these should be kebab-case: /testRuns -> /test-runs